app/controllers/account/billing-info/invoices.js

```diff
@@ -0,0 +1,34 @@
+import Controller from '@ember/controller';
+import { filterBy } from '@ember/object/computed';
+import moment from 'moment';
+import { computed } from '@ember/object';
+
+export default Controller.extend({
+  dueInvoices      : filterBy('model', 'status', 'due'),
+  paidInvoices     : filterBy('model', 'status', 'paid'),
+  upcomingInvoices : computed(function() {
```

Comment:
A computed property _must_ not return a promise. This should happen at the route level.

Reply:
@niranjan94 Moved it to the routes and returned a hash of promises